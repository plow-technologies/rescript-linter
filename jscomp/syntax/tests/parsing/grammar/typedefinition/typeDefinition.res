type t = string
type t = string and s = int
type t = string and s = int and u = float

type rec t = node<t>
type rec t = string and s = int
type rec t = string and s = int and u = float

