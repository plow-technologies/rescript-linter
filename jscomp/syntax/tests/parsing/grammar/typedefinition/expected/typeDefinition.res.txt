type nonrec t = string
type nonrec t = string
and s = int
type nonrec t = string
and s = int
and u = float
type t = t node
type t = string
and s = int
type t = string
and s = int
and u = float